### PR TITLE
Update NIRSpec TA Monitors to use Django DB Models

### PR DESCRIPTION
**Pull request by mfixstsci**

### CODE REVIEW

**Comment by mfixstsci** [comment]
@BradleySappington and @york-stsci I was able to verify using my generic user account that data was queried from the models and then stored when the monitor was completed 👍 ready for review at your convenience.

File: jwql/instrument_monitors/nirspec_monitors/ta_monitors/msata_monitor.py [hunk 1]
```diff
@@ -49,24 +48,42 @@
...
 from jwql.utils.utils import ensure_dir_exists, filesystem_path, get_config
 
+if not ON_GITHUB_ACTIONS and not ON_READTHEDOCS:
+    # Need to set up django apps before we can access the models
+    import django  # noqa: E402 (module level import not at top of file)
+
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "jwql.website.jwql_proj.settings")
+    django.setup()
+
+    from jwql.website.apps.jwql.monitor_models.ta import (
```
Comment (BradleySappington) [inline jwql/instrument_monitors/nirspec_monitors/ta_monitors/msata_monitor.py:76]: Is there a reason these non django imports are down here?

**Review by BradleySappington — COMMENTED** [review]
Not familiar enough with the monitors, so I'll need to go through this code with you to give a worthwhile review

**Comment by mfixstsci** [comment]
[af8f574](https://github.com/spacetelescope/jwql/pull/1499/commits/af8f57400b0249f6d5de56a71ed9821af06779c7) addresses #1171 @bhilbert4

File: jwql/instrument_monitors/nirspec_monitors/ta_monitors/msata_monitor.py (hunk 1 above)
Comment (mfixstsci) [reply jwql/instrument_monitors/nirspec_monitors/ta_monitors/msata_monitor.py:76]: I was following the example set in the readnoise monitor that @york-stsci wrote. I think that it has something to do with the fact that the imports are django models and you need to tell the module where to look for models and set them up first.

**Comment by mfixstsci** [comment]
Myself, @BradleySappington, @bhilbert4 and @york-stsci all reviewed this pull request over a phone call last week, merging now.

AFTER MERGE TO DO:
* [ ] Reset `NIRSpecTaQueryHistory` on `test`
* [ ]  Reset `NIRSpecTaQueryHistory` on `ops`

**Pull request merged**